### DISCUSSION TITLE
Default location for config.nu in Windows 10?

### DISCUSSION THREAD

**Discussion by gunungpw** [nushell/nushell#4827] [Q&A] [answered]
Hello new to nushell here, Thanks for maintaner an contributor for this great shell.
Where to find config file in nushell in windows 10? in docs i can't find reference for it 😢
Thanks sorry for my poor english, I still learning 😄

**Comment by kubouch** [comment, marked as answer]
Hi, you can check `$nu.config-path`.

**Reply by gunungpw** [reply]
Thank you
